lib/telegraf.js

```diff
@@ -119,7 +119,7 @@ class Telegraf extends Composer {
     }
     this.telegram.getUpdates(timeout, limit, offset, allowedUpdates)
       .catch((err) => {
-        const wait = err.retryAfter || this.options.retryAfter
+        const wait = err.parameters.retry_after || this.options.retryAfter
```

Comment:
I think we need to add an extra check:
`(err.parameters && err.parameters.retry_after) || this.options.retryAfter`